querydsl-sql/src/test/java/com/querydsl/sql/SelectBase.java

```diff
@@ -1265,6 +1265,15 @@ public void num_cast2() {
     }
 
     @Test
+    public void num_cast_dateTime() { //TODO: think of good name
```

Comment:
But it doesn't actually use cast anymore (that was the workaround).
It's now actually more of a `numeric_date_operation`.